js/arena.js

```diff
@@ -1,37 +1,51 @@
 'use strict';
 
-import Gladiator from './gladiator.js';
+// import Gladiator from './gladiator.js';
 
 export default class Arena {
 
+  // Capitalize the name of the arena and assign it. The arena has gladiators.
   constructor(name, gladiators = []) {
     const capitalizedName = name.charAt(0).toUpperCase() + name.slice(1);
     this.name = capitalizedName;
     this.gladiators = gladiators;
   }
 
+  // You can add a gladiator to the arena. You cannot exceed two gladiators in one arena
   addGladiator(gladiator) {
     (this.gladiators.length < 2) ? this.gladiators.push(gladiator) : false;
   }
 
+  // You can remove the eliminated gladiator based on their name.
+  removeGladiator(gladiator) {
```

Comment:
Maybe add a condition so that the function doesn't run if there are more than two gladiators in the Arena OR do a console.log that lets the user know that the third gladiator hasn't been added to the arena